features/networking/operator.feature

```diff
@@ -186,16 +186,21 @@ Feature: Operator related networking scenarios
     And the output should not contain "NotReady"
     Given the status of condition "Progressing" for network operator is :False
     """
+
   # @author anusaxen@redhat.com
   # @case_id OCP-24918
```

Comment:
http://pastebin.test.redhat.com/828146
Changed scenario a more smarter way included polling to check a partcular keyword in sdn logs to make sure flag is updated across the sdn and